### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by 1e4**

### CODE REVIEW

**Comment by 1e4** [comment]
Contains initial workflow for awesome lists along with a few items to start off with

File: README.md [hunk 1]
```diff
@@ -2,3 +2,23 @@
 
 > Curated list of awesomeness for Persistent Browser Based Games.
 
+[Resources](#resources)
+
+- [PBBG.com](https://pbbg.com) The home of PBBG's and text based browser games.
```
Comment (foohonpie) [inline README.md]: What do you think of simplifying to "The home of Persistent Browser-Based Games"?  That way we're not inferring a bias towards text-based games.

**Review by foohonpie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (1e4) [reply README.md]: Completely up to you, the text I added was just added on a whim to get going so I'm all for this change :)

**Comment by 1e4** [comment]
Updated with the new description

**Pull request merged**